src/pact.spec.ts

```diff
@@ -5,14 +5,14 @@ import fs = require("fs");
 import pact from "./pact";
 
 const expect = chai.expect;
-
 chai.use(chaiAsPromised);
+const currentDir = (process && process.mainModule) ? process.mainModule.filename : "";
 
 describe("Pact Spec", () => {
 	afterEach(() => pact.removeAllServers());
 
 	describe("Set Log Level", () => {
-		let originalLogLevel;
+		let originalLogLevel: any;
```

Comment:
string

Reply:
Actually it's a union type of `LogLevels` and `number | void`, as it gets use to both set and retrieve the log level - hence why I put any.